.github/workflows/PR_summary.yml

```diff
@@ -99,6 +99,9 @@ jobs:
         ## Technical debt changes
         techDebtVar="$(./scripts/technical-debt-metrics.sh pr_summary)"
 
-        message="$(printf '%s [%s](%s)%s\n\n%s\n\n---\n\n%s\n\n---\n\n%s\n' "${title}" "$(git rev-parse --short HEAD)" "${hashURL}" "${high_percentages}" "${importCount}" "${declDiff}" "${techDebtVar}")"
+        # store in a file, to avoid "long arguments" error.
+        printf '%s [%s](%s)%s\n\n%s\n\n---\n\n%s\n\n---\n\n%s\n' "${title}" "$(git rev-parse --short HEAD)" "${hashURL}" "${high_percentages}" "${importCount}" "${declDiff}" "${techDebtVar}" |
+          jq -Rs -n -c '{"body": inputs}' > messageFile.json
```

Comment:
I'd argue this line belongs in the other script, since it's that one that actually needs the json format.

Reply:
Just to be clear: you are saying that this script should store a text (`md`, really) file and the `jq` conversion should happen in the other file?

Reply:
Yes, that's my suggestion.

Reply:
I like this and I implemented it: the `workflow` file creates an `md` and then the script that posts the `json` file takes care of converting to `json` and posting.